index.html

```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="./style.css"></link>
+</head>
+<body>
+    <main>
+        <div class="title_intro">
+            <h1>freeCodeCamp Survey Form</h1>
+            <h3>Thank you for taking the time to help us improve this platform</h3>
+        </div>
+        
+        <form>
+            <label>
+                Name
+                <input type="text"/>
+            </label>
+            <label>
+                Email
+                <input type="email"/>
+            </label>
+            <label>
```

Comment:
instead of using `divs` and `labels`. its better to use `sections` ,`articles` and other elements for better semantics. check this link
[https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md](click here) for deeper explanation